build.mjs

```diff
@@ -100,15 +95,17 @@ try {
                                 },
                             },
                             env: {
-                                targets: "defaults",
+                                targets: "fully supports es6",
                                 include: [
-                                    "transform-classes",
-                                    "transform-arrow-functions",
```

Comment:
Keep this, hermes doesn't support async arrow functions, but it does support `async function() {}`